src/components/interface/SwapForm/components/Leverage/Leverage.tsx

```diff
@@ -22,6 +24,10 @@ export type LeverageProps = {
 }
 
 const Leverage = ({availableNotional, minMargin, notional, onChange, value, resetDeltaState}: LeverageProps) => {
+  const { sessionId } = useWallet();  
```

Comment:
remove these vars. probably some leftovers

Reply:
@ioanaroceanu2000 
Why was this resolved? 
Can you share the commit where this was addressed? 
If you have it locally push it please!

Best practice is to resolve the comment after you pushed the fix. The share it via message in the conversation. 

Example message: "@reviewer fixed in <commit_hash>"